app/controllers/admin/project_groups_controller.rb

```diff
@@ -14,7 +19,13 @@ def create
     authorize @entry
 
     @loan = @entry.project
-    @entry.parent = @loan.root_timeline_entry
+
+    parent_id = project_group_params[:parent_id]
+    if parent_id && parent_id.present?
```

Comment:
Don't think you need both of these expressions. .present? checks for nil? or empty?